docs/content/en/docs/install/cert-manager.md

```diff
@@ -27,73 +27,45 @@ The steps are:
 
 * Install the cert-manager of your choice
   if it is not already installed.
-* Modify the `Deployment` manifest of each KLT operator component.
-* Add the `Certificate` CRD for the cert-manager you are using.
-
-## Modify the KLT manifest
-
-You must modify the KLT manifest for each KLT operator component
-to make it aware of the cert-manager you are using.
-These instructions implement
-[cert-manager.io](https://cert-manager.io/);
-the process is similar for other cert-managers.
-
-To configure KLT to use your cert-manager,
-change the `Deployment` manifest of each KLT operator component
-and **replace** the following `volumes` definition
-
-   ```yaml
-   - emptyDir: {}
-     name: certs-dir
-   ```
-
-   with
-
-   ```yaml
-   - name: cert
-     secret:
-       defaultMode: 420
-       secretName: webhook-server-cert
-   ```
-
-Each manifest must have the following special annotation:
-
-```yaml
-cert-manager.io/inject-ca-from=klt-serving-cert/keptn-lifecycle-toolkit-system
-```
-
-The value of the annotation must match the
-`name/namespace` of the cert-manager CRD discussed below.
+* Add the `Certificate` and `Issuer` CRs for the cert-manager you are using.
+* Install KLT without `cert-manager` via helm
 
 ## Add the CRD for your cert-manager
 
-This is the CRD for `cert-manager.io`:
+These are the CRs for `cert-manager.io` to be applied to your cluster:
 
 ```yaml
 apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
-  name: klt-serving-cert 
-  namespace: keptn-lifecycle-toolkit-system
+  name: klt-certs
+  namespace: <your-klt-namespace>
```

Comment:
maybe $NAMESPACE would be more visible? Or we could even suggest a get namespace to save the env var and a command to substitute it and apply it?

Reply:
I guess this is enough, I mean it's just our good manner that we provide instructions how to run it with external cert-manager, if we already have ours in place